dvc/info.py

```diff
@@ -134,3 +142,26 @@ def _get_dvc_repo_info(self):
         return "dvc (subdir), git"
 
     return "dvc, git"
+
+
+def _get_external_remotes_cache(self):
+    cache_info = self.config.get("cache", {})
+
+    cache_vars = [
+        val for val in Schemes.__dict__.values() if isinstance(val, str)
+    ]
+    cache_key = list(set(cache_info.keys()).intersection(set(cache_vars)))
```

Comment:
This works, but the list of accepted external caches is more limited: https://github.com/iterative/dvc/blob/38dd1c9fe7d0418868cd3a2d0962df3dd49ead71/dvc/config.py#L146 .

Also, it is not easy to spot, but `dir` https://github.com/iterative/dvc/blob/38dd1c9fe7d0418868cd3a2d0962df3dd49ead71/dvc/config.py#L154 is also a special case of `local`.

Reply:
But isn't _local_ considered as external since it's not in the parent directory?

Reply:
Yeah, pretty much whenever cache dir is outside of repo it is considered external. Just saying that configuring `dir` here is kinda like creating a separate section and setting `cache.local`. Again, I know that this part is a bit confusing :slightly_smiling_face: